src/renderer/screens/manager/AppsList/Filter.js

```diff
@@ -9,33 +9,38 @@ import Box from "~/renderer/components/Box";
 import BoldToggle from "~/renderer/components/BoldToggle";
 import Text from "~/renderer/components/Text";
 
-const Filter = ({ onFilterChange, filter }: *) => {
+const Filter = ({ onFiltersChange, filters, t }: *) => {
   const onFilterChangeWrapper = useCallback(
     ({ selectedItem: item }) => {
-      if (!item) {
-        return;
+      if (!item) return;
+      const newFilters = [...filters];
+      const i = newFilters.indexOf(item.key);
+      if (i >= 0) {
+        newFilters.splice(i, 1);
+      } else {
+        newFilters.push(item.key);
       }
-      onFilterChange(item.key);
+      onFiltersChange(newFilters);
     },
-    [onFilterChange],
+    [filters, onFiltersChange],
   );
 
   const filterItems = [
-    {
-      key: "all",
-      label: <Trans i18nKey="manager.applist.filter.all" />,
-    },
-    {
-      key: "installed",
-      label: <Trans i18nKey="manager.applist.filter.installed" />,
-    },
+    // {
+    //   key: "all",
+    //   label: <Trans i18nKey="manager.applist.filter.all" />,
+    // },
+    // {
+    //   key: "installed",
+    //   label: <Trans i18nKey="manager.applist.filter.installed" />,
+    // },
     {
       key: "not_installed",
-      label: <Trans i18nKey="manager.applist.filter.notInstalled" />,
+      label: t("manager.applist.filter.not_installed"),
     },
     {
       key: "supported",
-      label: <Trans i18nKey="manager.applist.filter.supported" />,
+      label: t("manager.applist.filter.supported"),
```

Comment:
I'm kinda in favor of `<Trans` versions as much as possible. This allow to prevent having to connect on root components but prefer to connect on leafs.
even tho i've not really measure the perf of this, we can typically remove the nesting of `withTranslation(` in many cases.
maybe a topic we need to discuss more with everyone.

Reply:
I think I used the function instead of Trans in order to join the selected filters strings a bit further in the component. I can try and do it another way with Trans though.

Reply:
mmh that's weird, translations must not be used for anything else but something on rendering